.github/workflows/lint.yml

```diff
@@ -38,17 +38,17 @@ jobs:
     name: ament_${{ matrix.linter }}
     runs-on: ubuntu-latest
     container:
-      image: rostooling/setup-ros-docker:ubuntu-focal-ros-rolling-ros-base-latest
+      image: rostooling/setup-ros-docker:ubuntu-jammy-ros-humble-ros-base-latest
     strategy:
       fail-fast: false
       matrix:
-          linter: [cppcheck, cpplint, uncrustify]
```

Comment:
Why cppcheck needs to be removed from here?

Reply:
It's because cppcheck always fails on Jammy/Humble.  The problem is that cppcheck 2.X is *very* slow, so slow that it makes our CI times ridiculous.  So starting with Ubuntu 22.04, we disabled it, and we no longer run it in CI (honestly we should probably just remove it, but that's a different change).